api/v1/components/controller.go

```diff
@@ -23,5 +23,8 @@ type Controller struct {
 	Sidecars map[string]*Sidecar `json:"sidecars,omitempty"`
 	// +nullable
 	// +optional
-	Resources *ResourceRequirements `json:"resources,omitempty"`
+	Resources   *ResourceRequirements `json:"resources,omitempty"`
+	FastTimeout string                `json:"fastTimeout,omitempty"`
```

Comment:
There is a naming convention. https://github.com/zecke/Kubernetes/blob/master/docs/devel/api-conventions.md#naming-conventions

Reply:
Refactored with RateLimiter args names from [go-client](https://github.com/kubernetes/client-go/blob/8f44946f6cbe967fbe2e2548e76987680a89428e/util/workqueue/default_rate_limiters.go#L136)